deeplabcut/pose_estimation_tensorflow/training.py

```diff
@@ -190,7 +194,54 @@ def train_network(
             os.environ["CUDA_VISIBLE_DEVICES"] = str(gputouse)
     try:
         cfg_dlc = auxiliaryfunctions.read_plainconfig(poseconfigfile)
-        if "multi-animal" in cfg_dlc["dataset_type"]:
+
+
+        if superanimal_name != "":
+            from deeplabcut.modelzoo.utils import parse_available_supermodels
+            from dlclibrary.dlcmodelzoo.modelzoo_download import (
+                download_huggingface_model,
+                MODELOPTIONS,
+            )
+            import glob
+            dlc_root_path = auxiliaryfunctions.get_deeplabcut_path()
+            supermodels = parse_available_supermodels()    
+            weight_folder = str(
+                Path(dlc_root_path)
+                / "pose_estimation_tensorflow"
+                / "models"
+                / "pretrained"
+                / (superanimal_name + "_weights")
+            )    
+
+            if superanimal_name in MODELOPTIONS:
+                if not os.path.exists(weight_folder):
+                    download_huggingface_model(superanimal_name, weight_folder)
+                else:
+                    print(f"{weight_folder} exists, using the downloaded weights")
+            else:
+                print(f"{superanimal_name} not available. Available ones are: ", MODELOPTIONS)
```

Comment:
We could move the imports higher up, and test early whether `superanimal_name` is in `MODELOPTIONS`.